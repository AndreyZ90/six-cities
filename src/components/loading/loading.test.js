import React from 'react';
import TestRenderer from 'react-test-renderer';

import Loading from '@/components/loading/loading';

describe(`Loading component snapshot`, () => {
  test(`Should correctly render Loading component`, () => {
    const tree = TestRenderer.create(<Loading/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
