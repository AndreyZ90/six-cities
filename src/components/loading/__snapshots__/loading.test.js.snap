// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component snapshot Should correctly render Loading component 1`] = `
<div
  className="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
